static/css/style.css

```diff
@@ -299,6 +336,58 @@ h1.table_titles{
     z-index: 1021;
 }
 
+.team-card {
+    min-height: 380px;
+}
+
+.card-title .team-name:after {
+    content: '';
+    display: block;
+    height: 2px;
+    width: 60%;
+    margin: 0.5rem auto;
+    background: var(--s-grey);
+}
+
+.card-text-title {
+    font-size: 1.2rem;
+    font-style: italic;
+}
+
+.user-github-icon .fab {
+    font-size: 1.2rem;
+    margin: 0 0.5rem 0 0;
+    color: var(--p-navy);
+    transition: transform .4s;
+}
+
+.pagination {
+    margin: 1rem 0 2rem;
+    justify-content: center;
+}
+
+.dropdown-menu.judge-dropdown {
+    width: 100%;
+    border: 3px solid var(--p-orange);
+}
+
+.dropdown-menu.judge-dropdown .dropdown-item:hover {
+    width: 100%;
+    background: var(--p-orange);
+    color: white;
+    border-radius: 0px;
+}
+
+.card-body.text-left.hackathon-detailed-items {
+    padding: 2rem;
+}
```

Comment:
Missing formatting blank-line after this.

Reply:
Added blank line